src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -73,12 +73,12 @@ class UserService(protected val userInfo: UserInfo, dataSource: SlickDataSource,
     case GetRefreshTokenDate => getRefreshTokenDate() pipeTo sender
 
     case CreateUser => createUser() pipeTo sender
-    case AdminGetUserStatus(userRef) => adminGetUserStatus(userRef) pipeTo sender
-    case UserGetUserStatus => userGetUserStatus() pipeTo sender
-    case EnableUser(userRef) => enableUser(userRef) pipeTo sender
-    case DisableUser(userRef) => disableUser(userRef) pipeTo sender
-    case ListUsers => listUsers pipeTo sender
-    case ImportUsers(rawlsUserInfoList) => importUsers(rawlsUserInfoList) pipeTo sender
+    case AdminGetUserStatus(userRef) => asAdmin { getUserStatus(userRef) } pipeTo sender
```

Comment:
I like it up top and think we should try to move it all there at some point. I think it will be more clear and it will be easy to see which end points are missing security.